src/oc-client.js

```diff
@@ -328,21 +328,22 @@ var oc = oc || {};
         };
       };
 
-      var wasDollarThereAlready = !!$window.$;
-      oc.require('jQuery', JQUERY_URL, function(jQuery) {
-        requirePolyfills(jQuery, function() {
-          if (wasDollarThereAlready) {
-            // jQuery was already there. The client shares the same instance.
-            oc.$ = jQuery;
-          } else {
-            // jQuery wasn't there. The client dynamically downloads it and
-            // it tries to avoid sharing it by freeing the $ symbol.
-            oc.$ = jQuery.noConflict();
-          }
+      var wasJqueryThereAlready = !!$window.jQuery || !!$window.$;
 
+      if (wasJqueryThereAlready) {
+        var instanceOfJquery = $window.jQuery || $window.$;
+        requirePolyfills(instanceOfJquery, function() {
+          oc.$ = instanceOfJquery;
           done();
         });
-      });
+      } else {
+        oc.require('jQuery', JQUERY_URL, function(jQuery) {
```

Comment:
I'm not sure we need an `if/else` as `oc.require` already does some of [this logic](https://github.com/opencomponents/oc-client-browser/blob/master/src/oc-client.js#L208-L215), meaning that it won't fetch `jQuery` if already available.

So perhaps is enough to simply extend the `wasJqueryThereAlready` check as you suggested while leaving the rest of the logic un touched? What you think @debopamsengupta ?